src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class AbstractFigure implements Figure {
```

Comment:
you can implement calculable and drawable directly here, having an empty figure interface is redundant given you already have a Figure class

Reply:
not fixed